package.json

```diff
@@ -15,48 +15,58 @@
     "testWatch": "jest src/* --watch",
     "precommit": "make verify && node_modules/.bin/secret-squirrel",
     "prepush": "make unit-test && make verify -j3",
-    "commitmsg": "node_modules/.bin/secret-squirrel-commitmsg"
+    "commitmsg": "node_modules/.bin/secret-squirrel-commitmsg",
+    "postinstall": "rm -rf dist && mkdir dist && babel -d ./dist ./main.js --presets=es2015,react && babel -d ./dist/src ./src --presets=es2015,react"
   },
   "engines": {
     "node": "^6.2.0"
   },
   "author": "ciprian.lujeru@ft.com",
   "license": "MIT",
   "devDependencies": {
-    "@financial-times/n-gage": "^1.8.21",
-    "babel-cli": "~6.9.0",
-    "babel-core": "~6.2.1",
-    "babel-loader": "~6.2.0",
-    "babel-preset-es2015": "~6.1.18",
-    "babel-preset-react": "~6.1.18",
-    "babel-register": "~6.9.0",
-    "enzyme": "~1.4.1",
-    "jest": "^20.0.1",
-    "react-addons-test-utils": "~0.14.7"
+    "@financial-times/n-ui": "^6.14.0",
+    "@financial-times/n-heroku-tools": "^6.19.4",
+    "babel-cli": "^6.9.0",
+    "babel-core": "^6.2.1",
+    "babel-loader": "^6.2.0",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-es2015": "^6.1.18",
+    "babel-preset-react": "^6.1.18",
+    "babel-register": "^6.9.0",
+    "bower": "^1.7.9",
+    "chai": "^3.5.0",
+    "chai-http": "^3.0.0",
+    "chai-jquery": "^2.0.0",
+    "enzyme": "^2.9.1",
+    "eslint-plugin-react": "^5.2.2",
+    "jest": "^20.0.0",
+    "jest-enzyme": "^3.1.0",
+    "jsdom": "^8.1.0",
+    "mocha": "^2.4.5",
+    "node-sass": "^3.8.0",
+    "origami-build-tools": "^5.5.3",
+    "react-test-renderer": "^15.6.1",
+    "redux-mock-store": "^1.2.3"
   },
   "dependencies": {
-    "node-sass": "~3.8.0",
-    "@financial-times/n-express": "~16.9.0",
-    "babel-preset-stage-1": "~6.1.18",
-    "es6-promise": "~3.2.1",
-    "react": "~0.14.3",
-    "react-dom": "~0.14.3",
-    "react-router": "~2.0.1",
-    "polyfill-service": "~3.7.0",
-    "dotenv": "~2.0.0",
-    "origami-build-tools": "~5.6.1",
-    "hbs": "~4.0.0",
-    "bower": "~1.7.9",
-    "gulp": "~3.9.1",
-    "gulp-imagemin": "~2.2.1",
-    "gulp-livereload": "~3.8.1",
-    "gulp-nodemon": "~2.2.1",
-    "gulp-size": "~1.2.1",
-    "eslint-plugin-react": "~5.2.2",
-    "redux": "~3.5.2",
-    "react-redux": "~4.4.5",
-    "redux-logger": "~2.6.1",
-    "redux-thunk": "~2.1.0"
+    "babel-preset-stage-1": "^6.1.18",
+    "@financial-times/n-notification": "^6.0.0",
+    "dotenv": "^2.0.0",
+    "es6-promise": "^3.2.1",
+    "gulp": "^3.9.1",
+    "gulp-imagemin": "^2.2.1",
+    "gulp-livereload": "^3.8.1",
+    "gulp-nodemon": "^2.2.1",
+    "gulp-size": "^1.2.1",
```

Comment:
All these gulp deps should be dev dependencies

Reply:
https://trello.com/c/bboslv83/391-clean-dependencies-in-kmt-header